code/src/components/TaskCreator.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import uniqid from 'uniqid';
+import styled from 'styled-components';
+
+import tasks from '../reducers/tasks';
+
+const Form = styled.form`
+width: 100%;
+padding: 20px 0;
+margin: 10px 0;
+display: flex;
+flex-direction: row;
+justify-content: space-evenly;
+background-color: white;
+border-radius: 15px;
+box-shadow: 0  0 10px #aaaaaa;
+@media (min-width: 768px){
+  width: 680px;
+}
```

Comment:
Why you have no indentation in your style-components?

Reply:
Fixed. Did not know how the should look.